src/app/match-viewer-app/team-display/team-display.component.scss

```diff
@@ -1,9 +1,13 @@
 :host {
-    .team-display{
-      padding: 12px;
-      background-color: #5072A8;
-    }
-    .bans-text{
-      font-weight: 600;
-    }
+  .team-display{
+    padding: 12px;
+    background-color: #5072A8;
+  }
+  .bans-text{
```

Comment:
I would suggest to put the `.bans-text` and `.banned-champion-icon` class  inside `.team-display`.  The class structuring, or style structuring, in `*.scss` is based on the child to parent relationship in the html, I think ... lol. That's how I did it for summoner-tile and it allowed me to do certain stuff that was not possible if I were to use `*css`.  Check the `summoner-tile.scss` file along with the respective `html` file to see the changes.